test/e2e/websocket_test.go

```diff
@@ -67,9 +68,13 @@ func connect(t *testing.T, ingressIP string, domain string) (*websocket.Conn, er
 }
 
 func validateWebSocketConnection(t *testing.T, clients *test.Clients, names test.ResourceNames) error {
-	gatewayIP, err := ingress.GetIngressEndpoint(clients.KubeClient.Kube)
-	if err != nil {
-		return err
+	var err error
```

Comment:
nit: The error is only needed inside the `if` I think.

Reply:
that's correct, done